media/css/firefox/nothing-personal/_primary-cta.scss

```diff
@@ -0,0 +1,79 @@
+
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+// -------------------- CTA logic ----------------------- //
+.mzp-c-button.is-not-default, .mzp-c-button.is-default {
+    display: none;
+}
+
+html.is-firefox {
+    #protocol-nav-download-firefox {
+        display: none;
+    }
+
+    &.set-default-supported {
+        .mzp-c-button.is-not-default {
+            display: block;
+        }
+
+        .mzp-c-button.is-default {
+            display: none;
+        }
+
+
+        &.is-firefox-default {
+            .mzp-c-button.is-default {
+                display: block;
+            }
+
+            .mzp-c-button.is-not-default {
+                display: none;
+            }
+        }
+    }
+} // -------------------- end CTA logic ----------------------- //
+
+// -------------------- CTA styles ----------------------- //
+
+.c-page-header-inner { /* stylelint-disable no-duplicate-selectors  */
+    #protocol-nav-download-firefox {
+        margin-bottom: 0;
+
+        small {
+            display: none;
+        }
+    }
+
+    .mzp-c-button,
+    #protocol-nav-download-firefox > .mzp-c-button {
```

Comment:
**issue:** If I'm on an unsupported OS version this CTA is readable OK, but the button text could use a color contrast tweak:

<img width="1595" alt="image" src="https://github.com/user-attachments/assets/e1375aca-9e1a-4945-bbbb-b43456ac24c3">

Reply:
Good callout again! I'll just update the styles here to be more accessible 👍🏼